gef-ui/server/controllers/amendments/what-needs-to-change/validation.test.ts

```diff
@@ -0,0 +1,78 @@
+import { validateWhatNeedsToChange } from './validation';
+
+describe('validateWhatNeedsToChange', () => {
+  const errorTestCases = [
```

Comment:
What will happen if the properties (changeCoverEndDate and changeFacilityValue) simply don't exist can we add an assertion for the case please.

Reply:
We are always passing in values for this, if they don't exist they will each be received by this validator as `undefined`. It's not possible to call this function without passing these values in.